tools/apprunner-gradle-plugin/build.gradle

```diff
@@ -4,7 +4,22 @@ plugins {
   id 'com.gradle.plugin-publish' version '0.12.0'
 }
 
-version = file('version.txt').text.trim()
+def quarkusVersionBuffer = new ByteArrayOutputStream()
+exec {
+  commandLine = ['../../mvnw', 'help:evaluate', '-Dexpression=quarkus.version', '-q', '-DforceStdout', '-f', '../../pom.xml']
+  standardOutput = quarkusVersionBuffer
+}.assertNormalExitValue()
+
+if (!file('version.txt').exists()) {
```

Comment:
I would be tempted to just remove the `version.txt` business. I think the module below is supposed to do it but if we are already doing it for quarkus we might as well do it for the nessie version